qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -226,6 +226,11 @@ def setup(self):
         self.create_actions()
         self.populate_menus()
 
+        # turn on algorithm factory notifications
```

Comment:
Could we move this to the `post_mantid_init` function below? It keeps the code regarding the algorithm selector together as that contains a refresh call.